src/room.py

```diff
@@ -1,2 +1,40 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+# Implement a class to hold room information. This should have name and
+# description attributes.
+
+
+class Room:
+	def __init__ (self, name, description):
+		self.name = name
+		self.description = description
+		self.n_to = None
+		self.e_to = None
+		self.s_to = None
+		self.w_to = None
+		self.item_list = ['gold', 'wine', 'fire']
+
+	def move_to (self, direction):
```

Comment:
good opportunity to control input here by using `lower()` and `slice()` for input commands like "N" or "North"